modules/monitoring-accessing-the-metrics-targets-page.adoc

```diff
@@ -4,16 +4,59 @@
 
 :_content-type: PROCEDURE
 [id="monitoring-accessing-the-metrics-targets-page_{context}"]
-= Accessing the Metrics Targets page in the Administrator perspective
+= Accessing the Metrics targets page in the Administrator perspective
 
-You can view the *Metrics Targets* page in the *Administrator* perspective in the {product-title} web console.
+You can view the *Metrics targets* page in the *Administrator* perspective in the {product-title} web console.
 
 .Prerequisites
 
 * You have access to the cluster as an administrator for the project for which you want to view metrics targets.
 
 .Procedure
 
-* In the *Administrator* perspective, select *Observe* -> *Targets*. 
-The *Metrics Targets* page opens with a list of all service endpoint targets that are being scraped for metrics.
+. In the *Administrator* perspective, select *Observe* -> *Targets*. The *Metrics targets* page opens with a list of all service endpoint targets that are being scraped for metrics.
++
+--
+This page shows details about targets for default {product-title} and user-defined projects. This page lists the following information for each target:
 
+* Service endpoint URL being scraped
+* ServiceMonitor component being monitored
+* The **up** or **down** status of the target
+* Namespace
+* Last scrape time
+* Duration of the last scrape
+--
+
+. (Optional) The list of metrics targets can be long. To find a specific target, do any of the following:
++
+|===
+|To... |Do this...
+
+|Filter the targets by status and source. 
+a|Select filters in the *Filter* list.
+
+The following filtering options are available:
+
+* **Status** filters:
+** **Up**. The target is currently up and being actively scraped for metrics.
+** **Down**. The target is currently down and not being scraped for metrics.
+
+* **Source** filters:
+** **Platform**. Platform-level targets relate only to default Red Hat OpenShift Service on AWS projects. These projects provide core Red Hat OpenShift Service on AWS functionality.
+** **User**. User targets relate to user-defined projects. These projects are user-created and can be customized.
+
+|Search for a target by name or label. |Enter a search term in the **Text** or **Label** field next to the search box.
+
+|Sort the targets. |Click one or more of the **Endpoint Status**, **Namespace**, **Last Scrape**, and **Scrape Duration** column headers.
+|===
+
+. Click the URL in the **Endpoint** column for a target to navigate to its **Target details** page. This page provides information about the target, including:
++
+--
+** The endpoint URL being scraped for metrics
+** The current *Up* or *Down* status of the target
+** A link to the namespace
+** A link to the ServiceMonitor details
+** Labels attached to the target
+** The most recent time that the target was scraped for metrics
+--
```

Comment:
Since this procedure has been replaced by "monitoring-getting-detailed-information-about-a-target", it is rendered here: https://60762--docspreview.netlify.app/openshift-rosa/latest/monitoring/managing-metrics.html#getting-detailed-information-about-a-target_managing-metrics